nix/packages/stub.nix

```diff
@@ -0,0 +1,61 @@
+{ nixpkgs
+, stdenv
+, makeRustPlatform
+, hostPlatform
+, targetPlatform
+, lib
+, runCommand
+, fetchurl
+, clippy
+, path
+, enableLint ? false
+}:
+let
+  arch = targetPlatform.qemuArch;
+  cross = import nixpkgs {
+    inherit (hostPlatform) system;
+    crossSystem = lib.systems.examples."${arch}-embedded" // {
```

Comment:
TODO: once the mentioned PR land, replace this by just using the proper example system.